tools/download_ctftools

```diff
@@ -6,63 +6,71 @@
 #
 
 #
-# Copyright 2019 Joyent, Inc.
+# Copyright 2020 Joyent, Inc.
 #
 
 #
-# This program will download a prebuilt copy of the CTF tools from Manta.
-# These tools have been built using a technique to ensure they can be run on a
-# sufficiently old platform.  More details are available at:
-#
-#	https://github.com/jclulow/ctftools
+# This program will download a prebuilt copy of the CTF tools from Manta
+# corresponding the min_platform. The tools are built and uploaded via a Jenkins
+# job.
 #
 
-MANTA_URL='https://us-east.manta.joyent.com'
-BASE_PATH='/Joyent_Dev/public/ctftools'
-TARBALL='ctftools.20141030T081701Z.9543159.tar.gz'
+MANTA_HOST="https://us-east.manta.joyent.com"
+MIN_PLATFORM="$1"
+MPATH="/Joyent_Dev/public/builds/SmartOS/ctftools/master/built.joyent_$MIN_PLATFORM"
+TARBALL="ctftools.tar.gz"
 
 if [[ -z $1 ]]; then
-	printf 'ERROR: usage: download_ctftools <destination dir>\n' 2>&1
+	printf "ERROR: usage: download_ctftools <destination dir>\n" 2>&1
```

Comment:
We should probably update the usage message to include in the MIN_PLATFORM argument, and check for exactly two arguments.